docker/pulumi/Dockerfile

```diff
@@ -162,7 +162,7 @@ RUN fnm install 20 && \
   fnm alias 24 default
 ENV PATH=/usr/local/share/fnm/aliases/default/bin:$PATH
 RUN npm install -g corepack && \
-    corepack install -g pnpm yarn
+    corepack install -g pnpm yarn@1.22.22
```

Comment:
Is there a way to install any 1.x so we automatically pick up newer versions of classic?